R/simulation.R

```diff
@@ -21,17 +21,11 @@ Simulation <- R6::R6Class(
     path = function(value) {
       private$readOnlyProperty("path", value, self$root$path)
     },
-    #' @field settings SimulationSettings object for the simulation (read-only)
-    settings = function(value) {

```

Comment:
@Yuri05 I have removed this completely since it seems to cause more trouble. It's for internal use only